core/src/main/java/org/pentaho/platform/web/http/api/resources/GenericFileResource.java

```diff
@@ -54,24 +54,37 @@ public GenericFileResource( @NonNull IGenericFileService genericFileService ) {
   }
 
   @GET
-  @Path( "/folders/tree" )
+  @Path( "/files/tree" )
```

Comment:
Again, I think `files/` is not needed. The base path, `/generic-files`, already introduces the general "files or folders" idea, and the new `TreeFilter` enum accounts for the intended effect.

Reply:
Yes I like that better. I have gone ahead and changed all `generic-files/files/` endpoints to `generic-files/`